src/services/storage/drivers/redis/index.ts

```diff
@@ -0,0 +1,138 @@
+import IORedis from 'ioredis'
+import { StorageDriver } from '../../types'
+import { RedisStorageConfig } from '../../types/config'
+import { StoredUrl, UrlRequestData, UrlWithInformation } from '../../types/url'
+import cryptoRandomString from 'crypto-random-string'
+import { getConfig } from '../../../../config'
```

Comment:
You shouldn't import getConfig. Every configuration you need should already be in RedisStorageConfig